frontend/plugin-manager/OBSPluginManager.hpp

```diff
@@ -0,0 +1,19 @@
+#pragma once
+#include "ui_OBSPluginManager.h"
+
+#include <QDialog>
+#include <QWidget>
+#include <obs-module.h>
+
+#include "../widgets/OBSBasic.hpp"
+
+
+class OBSPluginManager : public QDialog {
+	Q_OBJECT
+	std::unique_ptr<Ui::OBSPluginManager> ui;
+public:
+	explicit OBSPluginManager(std::vector<OBSModuleInfo> const& modules, QWidget* parent = nullptr);
```

Comment:
Depending on use case, would it be beneficial to also have a constructor that takes an rvalue reference of a `std::vector<OBSModuleInfo>`?

Or are there no occasions where the vector is passed into the constructor and never used after? Because that'd usually be the case where I'd do something like
```C++
// Set up vector
OBSPluginManager manager{std::move(modules)};
// Continue to only work through the manager's API
```
Otherwise the constructor will always copy the contents of the reference into the vector created via the initialiser list.